elide-annotations/src/main/java/com/yahoo/elide/security/ChangeSpec.java

```diff
@@ -21,9 +21,17 @@
     @Override
     public String toString() {
         return String.format("ChangeSpec { resource=%s, field=%s, original=%s, modified=%s}",
-                resource,
+                safe(resource),
                 fieldName,
-                original,
-                modified);
+                safe(original),
+                safe(modified));
+    }
+
+    private String safe(Object object) {
+        try {
+            return String.valueOf(object);
+        } catch (Exception e) {
+            return e.toString();
```

Comment:
Maybe indicate that the error is related to _displaying_ the failure (and not the actual cause of failure).

Reply:
Actually, this will show up within the _field_ section of the debug log. What we have here already is good 👍